ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -10,7 +10,7 @@
 def _render_template(template_name: str, **template_args) -> str:
     file_path = str(pathlib.Path(__file__).parent / "templates" / template_name)
 
-    with open(file_path, "r") as file:
+    with open(file_path, "r",encoding='UTF-8') as file:
```

Comment:
Style: 
- [Black](https://github.com/psf/black) will change these single quotes to double quotes. `poetry run ni-python-styleguide fix` will run Black for you.
- Encoding names are case-insensitive, but the Python documentation uses lowercase: `encoding="utf-8"`

Reply:
Sorry for not running black because of [this](https://github.com/ni/measurementlink-python/pull/240#discussion_r1164952314). I guess your suggestion is:encoding="utf-8"?